SQL/0000-00-00-schema.sql

```diff
@@ -72,6 +72,15 @@ CREATE TABLE `language` (
 INSERT INTO language (language_code, language_label) VALUES
     ('en-CA', 'English');
 
+CREATE TABLE `sex` (
+  `SexID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `Sex` varchar(255) NOT NULL,
```

Comment:
Hhhhm, my previous comment was just a little thought I had during our meeting, but if I am to really review this pull request I guess I'll try to be more exhaustive.

After giving it more thought, I am still not fully satisfied with the field names. I think having `Sex` in these names when we are already in the `sex` table is a little redundant. For instance, I'd really prefer to write `sex.ID` and `sex.Name` rather than `sex.SexID` and `sex.Sex` in my SQL requests, especially since the first one may look like a foreign key and the second is quite cryptic (imo). So I strongly advocate for the names `ID` and `Name` for the field names.

Regarding capitalization, the database sure is messy but I think it would be a good idea to maintain a consistent convention in at least the new tables. I am under the impression that this convention is unofficially singular snake_case for the table names and PascalCase for the field names. I am fine with that and this PR matches this convention, but I am still new and this impression of mine may be wrong, so can someone with more database experience than me confirm or deny it ?

Finally, regarding the pointing of sex foreign keys on `ID` or `Name`, I do not have a strong opinion as both have their pros and cons. I think using `ID` as a foreign key would probably be conceptually cleaner, but also require to update the current database and queries as well as require more joins in the latter. Using `Name` is certainly easier and will lead to simpler queries.

It would probably have been easier if I had mentionned all that earlier so sorry for the late reply ! In any case, I'd like someone with more database experience than me to also review this pull request.

Reply:
ERROR 3780 (HY000) at line 155: Referencing column 'Sex' and referenced column 'Sex' in foreign key constraint 'FK_candidate_sex_1' are incompatible.

Reply:
> Regarding capitalization, the database sure is messy but I think it would be a good idea to maintain a consistent convention in at least the new tables. I am under the impression that this convention is unofficially singular snake_case for the table names and PascalCase for the field names. I am fine with that and this PR matches this convention, but I am still new and this impression of mine may be wrong, so can someone with more database experience than me confirm or deny it ?

That seems to match is the official policy documented in `docs/SQLModelingStandard.md` but it's so weakly enforced that I had to go look it up.

Reply:
Damn I looked at the documentation but somehow missed this file, thanks for pointing it to me. I forgot my glasses at home today so I guess I'm gonna use that as my excuse 😅